lilypad/server/services/prompts.py

```diff
@@ -15,13 +16,63 @@ class PromptService(BaseService[PromptTable, PromptCreate]):
     table: type[PromptTable] = PromptTable
     create_model: type[PromptCreate] = PromptCreate
 
+    def find_records_by_name(
```

Comment:
nit: should this be `find_prompts_by_name`?

Reply:
yeah i also updated another one like this